heart_disease/src/features/build_features.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import pandas as pd
+from sklearn.compose import ColumnTransformer
+from sklearn.impute import SimpleImputer
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import OneHotEncoder
+
+from src.entities.feature_params import FeatureParams, TransformerConfig
```

Comment:
Тоже relative import
```suggestion
from ..entities.feature_params import FeatureParams, TransformerConfig
```